about.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Digital Master Plan 2023</title>
+  <link rel="stylesheet" href="css/general.css">
+  <link rel="stylesheet" href="css/about.css">
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap');
+  </style>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+<body>
+  <my-menu></my-menu>
+  <section class="section d-flex-column d-ds-flex-column justify-content-center align-items-center container" id="about">
+    <div>
+      <h3 class="sub-title">
+        "Hello! Digital Kenya"
+      </h3>
+      
+      <h1 class="about-title">
+        digital master plan global summit 2023
+      </h1>
+  
+      <p class="para">
+        The global summit brings together the community of experts, academics, and activist who comprise the Creative Commons affliate network in a different country every two, This year, we hope to expand our invitation list including organizations and individuals.
+      </p>
+  
+      <div>
+        <p class="txt-details">
+          please contact us per any Email for any Further questions
+          about Digital Master Plan<br>
+          <span class="address-info">digital@info.ac.ke</span>
+        </p>
```

Comment:
- [ ] I would implore you to center this content, both on mobile and desktop versions so that it would have a close resemblance with the design template.

### Your Design

![image](https://user-images.githubusercontent.com/86200356/217480837-e0d1b1ac-fb2d-465d-97bc-76e2f68bd515.png)

### Template design
![image](https://user-images.githubusercontent.com/86200356/217480955-e2af1d73-f323-4145-bb9a-465817ef4aad.png)